helpers.go

```diff
@@ -59,6 +63,60 @@ func cmdWithFlags(cmd string, ctx *cli.Context, boolFlags []string) string {
 	return cmd
 }
 
+// This function clears a list of args (like the one provided by `os.Args`)
+// to match with some special cases of zypper.
+// For example:
+//   zypper lp --bugzilla
+// In the above case --buzilla acts as a boolean flag, while with:
+//   zypper lp --bugzilla=123
+// acts like a string flag.
+// We have to differentiate between invocations with and without the "=".
+// When the "=" is not found we have to artificially inject an empty string
+// to avoid the next parameter to be considered the flag value.
+func fixArgsForZypper(args []string) []string {
+	specialFlags := []string{
+		"-b", "--bugzilla",
+		"--cve",
+		"--issues",
+	}
+	sanitizedArgs := []string{}
```

Comment:
since sanitizedArgs can actually be bigger than the origin args slice, just ignore this note :)